ui/pages/confirmations/components/confirm/info/personal-sign/personal-sign.test.tsx

```diff
@@ -31,8 +29,7 @@ describe('personalSign', () => {
       },
     };
     const mockStore = configureMockStore([])(mockState);
-    const { queryByText } = renderWithProvider(<PersonalSign />, mockStore);
-
-    expect(queryByText('Request from')).not.toBeInTheDocument();
+    const { container } = renderWithProvider(<PersonalSignInfo />, mockStore);
+    expect(container).toMatchSnapshot();
```

Comment:
In case currentConfirmation or currentConfirmation.msgParams are falsy PersonalSignInfo will return null, resulting in an empty div snapshot. To improve readability should we use inline snapshot (toMatchInlineSnapshot) instead?